module_4/setup.py

```diff
@@ -0,0 +1,17 @@
+from setuptools import setup, find_packages
+
+
+setup(
```

Comment:
For the future. If there are dependencies, you should specify them in the `install_requires` section.